### DISCUSSION TITLE
Does biome support to write custom rules?

### DISCUSSION THREAD

**Discussion by loynoir** [biomejs/biome#2387] [Q&A]
For example, `typescript-eslint` support to write custom rules.

https://typescript-eslint.io/developers/custom-rules/

Does biome support to write custom rules?

https://biomejs.dev/linter/rules/

**Comment by ematipico** [comment]
No, it doesn't.

We're working towards plugins though. Check the discussions to get involved

**Comment by arendjr** [comment]
Not yet, we’re working on implementing a plugin system though: https://github.com/biomejs/biome/discussions/1762
